addons/web/static/src/utils/hooks.js

```diff
@@ -33,10 +34,9 @@ export function useBus(bus, eventName, callback) {
 export function useAutofocus(params = {}) {
   const comp = useComponent();
   // Prevent autofocus in mobile
-  // FIXME: device not yet available in the env
-  // if (comp.env.device.isMobile) {
-  //     return () => {};
-  // }
+  if (useService("ui").isSmall) {
```

Comment:
I think we have a shortcut for this in the env: `comp.env.isSmall`. We encourage to use it